web/html/src/manager/systems/all-list.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from "react";
+
+import { IconTag } from "components/icontag";
+import * as Systems from "components/systems";
+import { Column } from "components/table/Column";
+import { SearchField } from "components/table/SearchField";
+import { Table } from "components/table/Table";
+
+import { Utils } from "utils/functions";
+import Network from "utils/network";
+
+type Props = {
+  /** Locale of the help links */
+  docsLocale: string;
+  isAdmin: boolean;
+};
+
+const allListOptions = [
+  { value: "server_name", label: "System" },
+  { value: "system_kind", label: "System Kind" },
+  { value: "status_type", label: "Updates" },
+  { value: "total_errata_count", label: "Patches" },
+  { value: "outdated_packages", label: "Packages" },
+  { value: "extra_pkg_count", label: "Extra Packages" },
+  { value: "config_files_with_differences", label: "Config Diffs" },
+  { value: "channel_labels", label: "Base Channel" },
+  { value: "entitlement_level", label: "System Type" },
+  { value: "requires_reboot", label: "Requires Reboot" },
+  { value: "created_days", label: "Registered Days" },
+  { value: "group_count", label: "Groups" },
+];
+
+export function AllSystems(props: Props) {
+  const [selectedSystems, setSelectedSystems] = React.useState<String[]>([]);
+
+  const handleSelectedSystems = (items: String[]) => {
+    const removed = selectedSystems.filter((item) => !items.includes(item)).map((item) => [item, false]);
+    const added = items.filter((item) => !selectedSystems.includes(item)).map((item) => [item, true]);
+    const data = Object.assign({}, Object.fromEntries(added), Object.fromEntries(removed));
+    Network.post("/rhn/manager/api/sets/system_list", data);
```

Comment:
Would be polite to handle errors for this request.

Reply:
done